pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -52,9 +50,9 @@ def work(self):
                                 individual_defense = pokemon_data.get("individual_defense", 0)
 
                                 iv_display = '{}/{}/{}'.format(
-                                    individual_stamina,
                                     individual_attack,
-                                    individual_defense
+                                    individual_defense,
```

Comment:
its a nice feature to resort the Bot ordering of IV from SAD to ADS so its in the same order like the renaming.
It would be quite irritating if a pokemon appears with 4/10/15 (SAD) and then its renamed in 10/15/4 (ADS).
Therefore not change the order in general from SAD (why?) to a more logic on (ADS)